PWGHF/D2H/TableProducer/candidateCreatorB0Reduced.cxx

```diff
@@ -44,26 +46,33 @@ struct HfCandidateCreatorB0Reduced {
   Configurable<double> minParamChange{"minParamChange", 1.e-3, "stop iterations if largest change of any B0 is smaller than this"};
   Configurable<double> minRelChi2Change{"minRelChi2Change", 0.9, "stop iterations is chi2/chi2old > this"};
   // selection
-  Configurable<double> invMassWindowB0{"invMassWindowB0", 0.3, "invariant-mass window for B0 candidates"};
-
-  double massPi = RecoDecay::getMassPDG(kPiPlus);
-  double massD = RecoDecay::getMassPDG(pdg::Code::kDMinus);
-  double massB0 = RecoDecay::getMassPDG(pdg::Code::kB0);
-  double massDPi{0.};
+  Configurable<double> invMassWindowDPiTolerance{"invMassWindowDPiTolerance", 0.01, "invariant-mass window tolerance for DPi pair preselections (GeV/c2)"};
+  // variable that will store the value of invMassWindowB0 (defined in dataCreatorDplusPiReduced.cxx)
+  float myInvMassWindowDPi{1.};
+
+  // O2DatabasePDG service
+  Service<o2::framework::O2DatabasePDG> pdg;
+
+  double massPi{0.};
+  double massD{0.};
+  double massB0{0.};
+  double invMass2DPi{0.};
+  double invMass2DPiMin{0.};
+  double invMass2DPiMax{0.};
```

Comment:
These two variables can be declared directly in the `process` function, no need to have them as datamembers

Reply:
Hi @fgrosa ! Thanks for the feedback. I will apply your comment and transpose the fixes to B+ (I will try to do it by tomorrow).

Reply:
Also, now we should be able to remove the leptons columns of the PID table. For the moment (for B0 and B+) we only need Pi but should we also store K and Pr?

Reply:
Hi @AlexBigO that's great. I would only store the pion PID (keep in mind that already the pion one is not that powerful, since most of the tracks are pions anyway, so there is no need to have something complex for the PID of the bachelor)